docs/Klarna-Activation.md

```diff
@@ -22,29 +23,107 @@ Klarna library updates:
      
-     4.2 Ensure Klarna mandatory fields from the table above are populated, also note that **`telephonenumber`** is 
-     mandatory for Klarna, so ensure it is set in [`CartLike#billingAddress`](http://dev.commercetools.com/http-api-projects-carts.html#cart)[`#phone`](http://dev.commercetools.com/http-api-types.html#address)
-     before handling the cart/payment.
+     4.2 Note that **`telephonenumber`**,**`date of birth`** and **`IP`** is mandatory for Klarna, so ensure that they 
+     are set to the payment-object before triggering the payone payment authorisation
      
      4.3 Payone requires: **`billing and delivery address need to be identical`**
      
-## Klarna staging test
- 
-Use the following credentials to test Klarna payments on sandbox 
-https://developers.klarna.com/en/se/kpm/test-credentials
\ No newline at end of file
+# The Klarna payment process
+
+Enabling your customers to pay via Klarna requires you to integrate the Klarna payment method into your checkout process.
+In order to get an overview of the relationship and interaction on your checkout please see the following diagram:
+![](klarna-checkout-diagramm.png)
+## Start session request
+
+The first step is to set up a new checkout session by sending a genericpayment-request to Payone. This step is a 
+pre-condition for loading the Klarna widget. A session will remain valid for 48 hours after the last update.
+
+For creating this request, the payment-adapter offers the endpoint "/commercetools/start/session/", which expects 
+the Id of the payment object, for which the payment process should be started. When the session was started 
+successfully,  the "client token", which is needed to load the klarna-widget, is saved to the given payment object. 
+Additionally, the startsession-request to payone and the response are also populated to the payment object
+
+## Update Session
+
+This is currently not supported by the payment adapter. It`s recommended starting a new klarna session, when the 
+cart has changed.
+
+##  Load the klarna widget and authorize the payment
+
+Part of the response of the start session request is a client token, which is also saved is the payment object. This 
+token needs to be passed to the Klarna Widget, which sends the client token together with the selected payment method (e.g. "Klarna Pay Later") 
+to Klarna to get a authorization token. Please check the [payone documentation](https://docs.payone.com/display/public/PLATFORM/Special+remarks+-+Klarna+Payments#SpecialremarksKlarnaPayments-widget), how to integrate 
+the widget in your shop and authorize the payment against "Klarna".
+
+## Authorize payment within payone.
+
+After the Klarna widget was loaded and the payment authorization against klarna was successfully done, the 
+"authorization token", which is provided within response of the klarna authorization, must be added to the payment object. Then the 
```

Comment:
Perhaps we can show JSON-formatted code how authorization token is put into payment object?

Reply:
66f4d9ec5